examples/flutter_gallery/lib/demo/calculator/home.dart

```diff
@@ -162,7 +162,8 @@ class KeyPad extends StatelessWidget {
   Widget build(BuildContext context) {
     final ThemeData themeData = new ThemeData(
       primarySwatch: Colors.purple,
-      brightness: Brightness.dark
+      brightness: Brightness.dark,
+      platform: Theme.of(context).platform,
```

Comment:
(But in this case there is always a wrapping theme anyway because we know we're in a MaterialApp.)